dataprofiler/tests/labelers/test_integration_struct_data_labeler.py

```diff
@@ -113,8 +113,21 @@ def test_data_labeler_extend_labels(self):
             dp.labelers.StructuredDataLabeler._default_model_loc)
         data_labeler = dp.labelers.TrainableDataLabeler(dirpath=dirpath)
 
-        data_labeler.add_label('NEW_LABEL')
-
+        original_label_mapping = data_labeler.label_mapping.copy()
```

Comment:
still concerned that this test could create relative errors as we aren't being explicit

Reply:
but i'm not blocking approval